src/components/pages/doc/breadcrumbs/breadcrumbs.jsx

```diff
@@ -14,6 +14,17 @@ const Breadcrumbs = ({ breadcrumbs }) => (
       Docs
     </Link>
     <span>/</span>
+    {isPostgresPost && (
+      <>
+        <Link
+          className="transition-colors duration-200 hover:text-black dark:hover:text-white"
```

Comment:
It looks like the `className` is repeated four times in the component already. I'd defined it as `linkClasName` and reused it, wdyt?